lib/core/src/sdk.rs

```diff
@@ -44,9 +45,10 @@ use crate::{
     utils,
 };
 
-/// Claim tx feerate, in sats per vbyte.
-/// Since the  Liquid blocks are consistently empty for now, we hardcode the minimum feerate.
-pub const LIQUID_CLAIM_TX_FEERATE_MSAT: f32 = 100.0;
+/// The minimum acceptable fee rate when claiming using zero-conf
+pub const LIQUID_SDK_ZERO_CONF_FEE_RATE: f32 = 100.0;
```

Comment:
fee rate is usually expressed in sat per vbyte. Was the intention to be 0.1 sat per vbyte?

Reply:
Indeed, to avoid rounding errors, we represent it in msat/vbyte